airbyte-integrations/connectors/source-salesforce/integration_tests/bulk_error_test.py

```diff
@@ -87,6 +87,4 @@ def test_failed_jobs_with_successful_switching(caplog, input_sandbox_config, str
         m.register_uri("DELETE", job_matcher, json={})
         with caplog.at_level(logging.WARNING):
             loaded_record_ids = set(record["Id"] for record in stream.read_records(sync_mode=SyncMode.full_refresh))
-        for i, log_message in enumerate(log_messages, 1):
-            assert log_message in caplog.records[-i].message
```

Comment:
why is this assert not needed anymore?

Reply:
Returned back. Was lost during experiments.